aiidalab_widgets_base/process.py

```diff
@@ -102,13 +109,19 @@ def follow(self, detach=False):
             update_state.start()
         else:
             self._follow()
+            for func in self._run_after_completed:
+                func(self.process)
+
+    def run_after_completed(self, function):
```

Comment:
Did you not like my suggestion of following the `on_<event>` pattern for registering callbacks?

Reply:
I loved it :)